File: C#/12-Enumerações/Enumeracao.cs
using System;

class Enumeracao
{
    //Podemos usar enumerações para enumerar os meses
    //Para utilizar enumerações usamos:
    enum Mes {Janeiro, Fevereiro, Março, Abril, Maio, Junho,
              Julho, Agosto, Setembro, Outubro, Novembro, Dezembro};

    enum Dia {Domingo, Segunda, Terça, Quarta, Quinta, Sexta, Sábado};

    static void Main(string[] args)
    {
    //Toda ves que criarmos variaveis do tipo Mes, elas poderão usar os 
    //valores que estão na enumeração
        Mes primeiro = Mes.Janeiro;

        Dia quinto = Dia.Quinta;

        Console.WriteLine("Dia : {0} , Mês : {1}", quinto, primeiro);
        Console.ReadLine();

    }
}

File: Java/16-Enumerações/Enumeracao.java
//Enumerações devem ficar fora da classe
// Podemos usar enumerações para mostrar os meses
enum Mes {
    JANEIRO, FEVEREIRO, MARCO, ABRIL, MAIO, JUNHO, JULHO, AGOSTO, SETEMBRO, OUTUBRO, NOVEMBRO, DEZEMBRO
};

class Enumeracao {
    public static void main(String[] args) {
        // Agora podemos criar objetos de Mes e atribuir qualquer valor da enumeração
        Mes primeiro = Mes.JANEIRO;
        Mes ultimo = Mes.DEZEMBRO;

        System.out.println("O primeiro mes do ano e " + primeiro);
        System.out.println("O ultimo mes do ano e " + ultimo);
    }
}